engines/mxnet/mxnet-engine/build.gradle

```diff
@@ -32,7 +29,12 @@ pmdMain.source = 'src/main/java'
 
 tasks.register('jnarator') {
     dependsOn ":engines:mxnet:jnarator:jar"
+    finalizedBy verifyJava
```

Comment:
why is this a `finalizedBy` relationship and not a `dependsOn` from the `verifyJava`?

Reply:
Same reasoning as the other case

Reply:
Fixed this too, it is now a proper dependency